doc/source/getting_started/comparison/includes/if_then.rst

```diff
@@ -0,0 +1,12 @@
+The same operation in pandas can be accomplished using
```

Comment:
are any of these shared across compraisons? its not big deal, but you can simply re-use them.

Reply:
Sorry, what do you mean by "these"?

Reply:
oh it looks like you are doing what i thought (e.g. putting includes in a common location) and then using for comparison_excel/sas/r etc. which is great.

so nvm